templates/BuildMyCharm/config.xml

```diff
@@ -156,7 +166,25 @@ trap "${new_trap}" EXIT
 ARTIFACTS_DIR=/srv/artifacts/{{job_name}}/$BUILD_NUMBER
 mkdir -p $ARTIFACTS_DIR
 ln -s $ARTIFACTS_DIR /var/lib/jenkins/jobs/$JOB_NAME/builds/$BUILD_NUMBER/archive
-env MATRIX_OUTPUT_DIR=$ARTIFACTS_DIR cwr -F ${MODELS_TO_TEST} totest.yaml --results-dir /srv/artifacts --test-id $BUILD_NUMBER
+
+# In case output scenario is not set
+if [ -z ${OUTPUT_SCENARIO+x} ]; then
+    export OUTPUT_SCENARIO=""
+fi
```

Comment:
same comment as above - do we really need the above block to set an unset var to an empty string?